docs/integrations/web-servers/haproxy.md

```diff
@@ -361,7 +361,7 @@ Please keep the **port(514)** handy as we will use it in next steps.
     Follow the steps below to enable HAProxy logs to go to log files :
 
 
-1. You can enable HAProxy logs to syslog by adding the following line in the **global** section of **/etc/haproxy/haproxy.cfg **file. This means that HAProxy will send its messages to rsyslog on 127.0.0.1.
+1. You can enable HAProxy logs to syslog by adding the following line in the **global** section of **/etc/haproxy/haproxy.cfg** file. This means that HAProxy will send its messages to rsyslog on 127.0.0.1.
 
 
```

Comment:
Here's another numbered list that needs to have indenting fixed so that it numbers properly:
https://help.sumologic.com/docs/integrations/web-servers/haproxy/#configure-logs-collection

Reply:
thank you! I have another PR coming (in progress) specifically for indentation